README.md

```diff
@@ -1,5 +1,32 @@
 # Constitution
 
-The governing rules of Hackers at Cambridge.
+The governing rules of "Hackers at Cambridge", hereafter (and in the rest of the repository) referred to as the "Society".
 
-[Read the constitution here](constitution.md).
+[Read the constitution here](constitution.md). You can also find a web version
+[on our website](https://hackersatcambridge.com/constitution).
+
+# This Repository
+
+As laid out in the [constitution](constitution.md#society-repository), this repository is used to list all rules and guidelines of the Society that are not defined or contained in the constitution. This repository exists for the benefit of the executive committee, but is free to read for the general public for those who may be interested.
+
+# Our Website
+
+The official website of the Society is located at [https://hackersatcambridge.com](https://hackersatcambridge.com).
+
+# Making changes to this repository
+
+Any changes to this repository must require a majority vote at a meeting of the Executive Committee may issue a change to the Repository's documents. Please view [the template](.github/PULL_REQUEST_TEMPLATE.md) for more information.
```

Comment:
"Any changes to this repository must require a majority vote at a meeting of the Executive Committee may issue a change to the Repository's documents." Still doesn't make sense to me.
Does the following get across what you're trying to say?
"Any changes to this repository's documents require a majority approval vote at a meeting of the Executive Committee"